File: wrappers/java/src/main/java/com/evernym/vdrtools/payments/ExtraFundsException.java
package com.evernym.vdrtools.payments;

import com.evernym.vdrtools.ErrorCode;
import com.evernym.vdrtools.IndyException;

public class ExtraFundsException extends IndyException {
    private static final long serialVersionUID = 6397499268992083529L;
    private static final String message = "Extra funds on inputs";

    /**
     * Initializes a new {@link ExtraFundsException} with the specified message.
     */
    public ExtraFundsException() {
        super(message, ErrorCode.ExtraFundsError.value());
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/PaymentsApi/ExtraFundsException.cs
namespace Hyperledger.Indy.PaymentsApi
{
    /// <summary>
    /// Extra funds on inputs.
    /// </summary>
    public class ExtraFundsException : IndyException
    {
        internal ExtraFundsException(string message) : base(message, (int)ErrorCode.PaymentExtraFundsError)
        {
        }
    }
}
